src/main/native/cpp/analysis/AnalysisManager.cpp

```diff
@@ -24,10 +25,25 @@
 #include "sysid/analysis/AnalysisType.h"
 #include "sysid/analysis/FilteringUtils.h"
 #include "sysid/analysis/JSONConverter.h"
+#include "sysid/analysis/Storage.h"
 #include "sysid/analysis/TrackWidthAnalysis.h"
 
 using namespace sysid;
 
+template <typename T>
+void ApplyToData(
```

Comment:
The function arguments need documentation, including what `action` and `specifier` are for.